Update style-guide.md (#206)
medic/cht-docs: content/en/contribute/docs/style-guide.md

### Images

> The figure shortcode is preferred for images as it uses built-in styling that is responsive, however Markdown syntax is also acceptable. Do not use html styling.

Reviewer: ```suggestion
The [image markdown syntax](https://www.markdownguide.org/basic-syntax/#images-1) can be used for images, but if any styling is required use the built-in `figure` shortcode. With the `figure` shortcode [many fields are configurable](https://gohugo.io/content-management/shortcodes/#figure), and the position and size can be responsive with [Bootstrap grid classes](https://getbootstrap.com/docs/4.0/layout/grid/#responsive-classes). You should avoid using the HTML `img` tag in the documentation.
```

| Do | Don't |
|---|---|

> | `{{< figure src="image.png" class="right col-6 col-lg-3" >}}` | `<img src="image.png" width="30%" align="right">` |

Reviewer: ```suggestion
| `{{< figure src="image.png" class="right col-6 col-lg-3" >}}` | `<img src="image.png" width="30%" align="right">` |
| `{{< figure src="image.png" class="right col-6 col-lg-3" >}}` | `<img src="image.png" style="width:30%; align:right;">` |
```

> The image can also link to a file by using the link attribute.

Reviewer: ```suggestion
It is good practice for the image to link to the image file so that a larger version can be viewed easily. This can be done using the `link` attribute with the `figure` shortcode, which is less error prone than adding a link to the markdown image notation.
```

> ```

Reviewer: ```suggestion
```

> {{< figure src="image.png" link="image.png" class="right col-6 col-lg-3" >}}

Reviewer: ```suggestion
| Do | Don't |
|---|---|
| `{{< figure src="image.png" link="image.png" alt="Alt text" title="Image Title">}}` | `[![Alt text](image.png "Image Title")](image.png)` |
```

> ```

Reviewer: ```suggestion
```
